app/components/transUnit/TransUnitService.js

```diff
@@ -170,7 +198,26 @@
        updateSaveButton);
 
     function modifyTranslationText(phrase, newText) {
```

Comment:
I think this would be better named something like `setTranslationText`, since it is not making any reference to the current value.